Python/Instagram-Liker-Bot/Instagram_Liker_Bot.py

```diff
@@ -0,0 +1,48 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.common.keys import Keys
+from getpass import getpass
+import time
+
+chrome=webdriver.Chrome(ChromeDriverManager().install())
+chrome.get("https://www.instagram.com")
+time.sleep(10)
+
+username=chrome.find_element_by_xpath('//*[@id="loginForm"]/div/div[1]/div/label/input')
+username.send_keys(input("Enter your username")) #enter username
```

Comment:
Can we make a `.env` file for this input? Would be much safer and standard.

Reply:
Ok so look, here is what you need to do.

```
from decouple import config
```
make one .env file

It should contain your username and password.

Pass those values from the env file. The can be done using `dotenv` as well. Like [here](https://dev.to/biplov/handling-passwords-and-secret-keys-using-environment-variables-2ei0)

Finally there is a `getuser()` functionality as well just like `getpass()` can it be used?